index.html

```diff
@@ -3287,155 +3680,190 @@ <h5>Holder Binding Derived Proof</h5>
+            data-include-format="text"></pre>
+            <p>
+The |prover_nym| and |secretProverBlind| are retained by the holder and kept
+secret. The |commitmentWithProof| value needs to be communicated to the issuer.
+            </p>
+          </section>
+          <section>
+            <h5>Pseudonym Base Proof</h5>
+            <p>
+The addition of a base proof under the pseudonym feature option begins
+with the issuer receiving a |commitmentWithProof| value from the holder and
+generating a cryptographically random value for |signer_nym_entropy|.
+            </p>
+            <pre class="example nohighlight" title="Signer Nym Entropy"
+            data-include="TestVectors/FeatureInputs/signerNymEntropy.json"
+            data-include-format="text"></pre>
             <p>
-To issue a credential under the <em>pseudonym with issuer-known PID</em> feature,
-the issuer generates a cryptographically random |pid| value for a holder. This
-value is shared with the holder but is otherwise kept secret. This value is
-shown below.
-          </p>
-          <pre class="example nohighlight" title="Issuer Generated PID"
-          data-include="TestVectors/FeatureInputs/issuerPid.json"
-          data-include-format="text"></pre>
-          <p>
 This example will make use of the same key material as shown in
 <a href="#example-private-and-public-keys-for-signature-0"></a>,
 the same unsigned document as shown in
 <a href="#example-credential-without-proof-for-features"></a>, and the same
 mandatory pointers as shown in
 <a href="#example-mandatory-pointers-for-features"></a>. This results in the
-same canonical document as in
-<a href="#example-canonical-document-for-features"></a>, the same HMAC document
-as in
+same canonical document as shown in
+<a href="#example-canonical-document-for-features"></a>,
+the same canonical HMAC document as shown in
 <a href="#example-canonical-hmac-document-for-features"></a>, and the same "add
-base transformation" as in
+base transformation" as shown in
 <a href="#example-add-base-transformation-for-features"></a>.
           </p>
           <p>
-This example makes use of the same proof configuration as
+This example makes use of the same proof configuration as in
 <a href="#example-base-proof-configuration-for-features"></a>.
 This results in the same canonical base proof as in
 <a href="#example-canonical-base-proof-configuration-for-features"></a>.
 Combining this with the above assumptions leads to the same base hashes as in
 <a href="#example-add-base-hashes-for-features"></a>.
           </p>
           <p>
-Since |featureOption| is equal to `"pseudonym_issuer_pid"`, the procedure of
-section <a href="#base-proof-serialization-bbs-2023">Base Proof
-Serialization</a> will produce the output shown below. This makes use of the
+Since |featureOption| is equal to `"pseudonym"`, the procedure of
+section <a href="#base-proof-serialization-bbs-2023"></a> will produce the
```

Comment:
`Section <` and `section <` should be consistent throughout. I prefer the former; others may prefer the latter. Decision is necessary, and should flow through this and all documents.